boards/vendor/custom_plank/board.cmake

```
@@ -3,10 +3,12 @@
 
 board_runner_args(jlink "--device=nrf52" "--speed=4000")
 board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
+board_runner_args(example "--param=custom_plank")
```

I'd choose another parameter value, maybe

```suggestion
board_runner_args(example "--param=foo")
```